functions/Element/getElementPosition.yaml

```diff
@@ -0,0 +1,30 @@
+shared:
```

Comment:
You should consider **shared** to be a base, that you copy/merge to both **client** and **server** properties. Just do this:
```yaml
shared: &shared
  ...

server:
  <<: *shared

client:
  <<: *shared
```